asdf_astropy/converters/transform/tests/test_transform.py

```diff
@@ -402,15 +402,6 @@ def _iterate_model_classes():
         assert len(missing) == 0, message
 
 
-def test_legacy_const(tmpdir):
-    model = astropy_models.Const1D(amplitude=5.)
-    assert_model_roundtrip(model, tmpdir, version="1.3.0")
-
-    model = astropy_models.Const2D(amplitude=5.)
-    with pytest.raises(TypeError, match="does not support models with > 1 dimension"):
-        assert_model_roundtrip(model, tmpdir, version="1.3.0")
-
```

Comment:
Why is this test removed? Is it covered by another test?

Reply:
Now that I've removed the ASDF Standard version requirement from the transforms, asdf will use the latest version of the transform tags to write the file, which supports models with > 1 dimension.  It's not very easy to force asdf to use an earlier version of the transforms, which is what we'd need to do to test this error.  I suspect we'll need to add a new feature to the asdf config to support that.

Reply:
... I should xfail this, shouldn't I.  I'll make that change in a sec.